configdb/configdb.go

```diff
@@ -19,12 +19,12 @@ type ConfigDB struct {
 	DNSNameservers     map[string]DNSNameserver  `json:"DNS_NAMESERVER,omitempty"`
 	Features           map[string]Feature        `json:"FEATURE,omitempty"`
 	Interfaces         map[string]Interface      `json:"INTERFACE,omitempty"`
-	LLDP               LLDP                      `json:"LLDP"`
+	LLDP               *LLDP                     `json:"LLDP,omitempty"`
 	LoopbackInterface  map[string]struct{}       `json:"LOOPBACK_INTERFACE,omitempty"`
 	MCLAGDomains       map[string]MCLAGDomain    `json:"MCLAG_DOMAIN,omitempty"`
 	MCLAGInterfaces    map[string]MCLAGInterface `json:"MCLAG_INTERFACE,omitempty"`
 	MCLAGUniqueIPs     map[string]MCLAGUniqueIP  `json:"MCLAG_UNIQUE_IP,omitempty"`
-	MgmtInterfaces     map[string]MgmtInterface  `json:"MGMT_INTERFACE,omitempty"`
+	MgmtInterface      map[string]MgmtInterface  `json:"MGMT_INTERFACE,omitempty"`
```

Comment:
Even if the Key in the config_db is singular, it is in fact plural, so having it plural in the go code makes more sense for me.